src/components/LoadingIndicator/LoadingIndicator.spec.jsx

```diff
@@ -9,125 +9,6 @@ const { LoadingMessage } = LoadingIndicator;
 describe('LoadingIndicator', () => {
 	common(LoadingIndicator);
 
-	describe('isLoading', () => {
```

Comment:
Most of the tests for `LoadingIndicator` moved into `OverlayWrapper.spec.jsx`, since the functionality moved there.